galeracluster/source/monitoringthecluster.rst

```diff
@@ -1,305 +1,288 @@
+
+.. code-block:: mysql
+
+   SHOW GLOBAL STATUS LIKE 'wsrep_%';
+
+   +------------------------+-------+
+   | Variable_name          | Value |
+   +------------------------+-------+
+   | wsrep_protocol_version | 5     |
+   | wsrep_last_committed   | 202   |
+   | ...                    | ...   |
+   | wsrep_thread_count     | 2     |
+   +------------------------+-------+
+
+.. seealso:: In addition to checking status variables through the database client, you can also monitor for changes in cluster membership and node status through ``wsrep_notify_cmd.sh``.  For more information on its use, see :doc:`notificationcmd`.
+
+
+---------------------------------
+ Checking Cluster Integrity
+---------------------------------
+.. _`check-cluster-integrity`:
+
+.. index::
+   pair: Parameters; wsrep_cluster_state_uuid
+.. index::
+   pair: Parameters; wsrep_cluster_conf_id
+.. index::
+   pair: Parameters; wsrep_cluster_size
+.. index::
+   pair: Parameters; wsrep_cluster_status
+
+The cluster has integrity when all nodes in it receive and replicate write-sets from all other nodes.  The cluster begins to lose integrity when this breaks down, such as the cluster is down, become partitioned, or experiences a split-brain situation.
+
+You can check cluster integrity using the following status variables:
+
+- :ref:`wsrep_cluster_state_uuid <wsrep_cluster_state_uuid>` shows the cluster state UUID, which you can use to determine whether the node is part of the cluster.
+
+  .. code-block:: mysql
+
+     SHOW GLOBAL STATUS LIKE 'wsrep_cluster_state_uuid';
+
+     +--------------------------+--------------------------------------+
+     | Variable_name            | Value                                |
+     +--------------------------+--------------------------------------+
+     | wsrep_cluster_state_uuid | d6a51a3a-b378-11e4-924b-23b6ec126a13 |
+     +--------------------------+--------------------------------------+
+
+  Each node in the cluster should provide the same value.  When a node carries a different value, this indicates that it is no longer connected to rest of the cluster.
+
+- :ref:`wsrep_cluster_conf_id <wsrep_cluster_conf_id>` shows the total number of cluster changes happening, which you can use to determine whether or not the node is a part of the Primary Component.
+
+  .. code-block:: mysql
+
+     SHOW GLOBAL STATUS LIKE 'wsrep_cluster_conf_id';
+
+     +-----------------------+-------+
+     | Variable_name         | Value |
+     +-----------------------+-------+
+     | wsrep_cluster_conf_id | 32    |
+     +-----------------------+-------+
+
+  Each node in the cluster should provide the same value.  When a node carries a different, this indicates that the cluster is partitioned.  Once the node reestablish network connectivity, the value aligns itself with the others.
```

Comment:
a different value ...

once the node reestablishes

aligns itself with the other nodes